code/__DEFINES/misc.dm

```diff
@@ -434,3 +434,8 @@ GLOBAL_LIST_INIT(ghost_others_options, list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 
 //for obj explosion block calculation
 #define EXPLOSION_BLOCK_PROC -1
+
+//for determining which type of heartbeat sound is playing
+#define BEAT_FAST 1
+#define BEAT_SLOW 2
+#define BEAT_NONE 0
```

Comment:
whenever we have defines we always have a ```_NONE```, if they're flags, we also tend to have an ```_ALL``` (```~0```)